packages/dev/core/src/Buffers/buffer.ts

```diff
@@ -1026,25 +1026,16 @@ export class VertexBuffer {
                 return new Float32Array(data, byteOffset, count);
             } else {
                 const offset = data.byteOffset + byteOffset;
-                if (forceCopy) {
-                    const result = new Float32Array(count);
-                    const source = new Float32Array(data.buffer, offset, count);
-
+                const wrongAlign = (offset & 3) !== 0;
+                if (forceCopy || wrongAlign) {
+                    if(wrongAlign) console.warn("Float array must be aligned to 4-bytes border")

```

Comment:
nit: You can maybe structure the code like this to avoid adding another check:
```suggestion
                if ((offset & 3) !== 0) {
                    Logger.Warn("...");
                    forceCopy = true;
                }
                if (forceCopy) {
```